triage-actions/stale-closer/StaleCloser.test.ts

```diff
@@ -0,0 +1,247 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { expect } from 'chai'
+import { Comment } from '../api/api'
+import { Testbed, TestbedIssue, TestbedIssueConstructorArgs } from '../api/testbed'
+import { daysAgoToTimestamp } from '../common/utils'
+import { StaleCloser, STALE_MARKER, WARN_MARKER } from './StaleCloser'
+
+const initalizeTestbed = (
+	issueConfig: TestbedIssueConstructorArgs,
+	comments?: { body: string; daysAgo: number }[],
+): { testbed: Testbed; issueTestbed: TestbedIssue } => {
+	const issue = new TestbedIssue(
+		{},
+		{
+			issue: Object.assign(
+				{ createdAt: daysAgoToTimestamp(10), updatedAt: daysAgoToTimestamp(10) },
+				issueConfig.issue,
+			),
+			labels: issueConfig.labels,
+			upvotes: issueConfig.upvotes,
+			comments: (comments || []).map((comment, index) => ({
+				author: { name: 'rando' },
+				body: comment.body,
+				id: index,
+				timestamp: daysAgoToTimestamp(comment.daysAgo),
+			})),
+		},
+	)
+
+	return {
+		testbed: new Testbed({
+			queryRunner: async function* () {
+				yield [issue]
+			},
+		}),
+		issueTestbed: issue,
+	}
+}
+
+const getAllComments = async (issue: TestbedIssue): Promise<Comment[]> => {
+	const comments = []
+	for await (const page of issue.getComments()) comments.push(...page)
+	return comments
+}
+
+const testConfig: [number, string, number, string, number, number, string, string, string] = [
```

Comment:
Are there upsides to defining the `StaleCloser` constructor parameters like this instead of using an object? It's nice to have the property names to refer to like with [`FeatureRequestConfig`](https://github.com/microsoft/vscode-github-triage-actions/blob/cd7ec725801fe3107cc33cf9a1446f36441cee8b/feature-request/FeatureRequest.ts#L15). I could see it being cumbersome to refer to the `StaleCloser` constructor to see what these values mean.

Reply:
I mean...in this case it's just because I wrote StaleCloser.ts first with normal parameters (instead of an object) and when I got to the test file I was like "Oh an object would be better here, but meh it's just a test file"